src/sentry/api/bases/organization.py

```diff
@@ -394,18 +394,18 @@ def _filter_projects_by_permissions(
             if is_active_superuser(request) or include_all_accessible:
                 span.set_tag("mode", "has_project_access")
                 proj_filter = request.access.has_project_access
+            elif is_active_staff(request):
```

Comment:
I thought this check could be nested inside `elif not filter_by_membership:` but we actually use it in _admin for fetching all projects.